keepkeyctl

```diff
@@ -309,6 +309,11 @@ class Commands(object):
         address = self.client.cosmos_get_address(address_n, args.show_display)
         return address
 
+    def binance_get_address(self, args):
```

Comment:
Needs a `.help` like here:
https://github.com/keepkey/python-keepkey/pull/94/files#diff-65792a4bbabaa290f0fa80d3394a2cc0R477

And `.args` like here:
https://github.com/keepkey/python-keepkey/pull/94/files#diff-65792a4bbabaa290f0fa80d3394a2cc0R552